Sources/VCIClient/common/CborLibraryUtils.swift

```diff
@@ -0,0 +1,100 @@
+import Foundation
+import SwiftCBOR
+
+public class CborLibrayUtils{
+    public init() {}
+    
+    public func decodeAndParseMdoc(base64EncodedString: String) throws -> [String:Any]{
+        let logTag = Util.getLogTag(className: String(describing: type(of: self)))
+        guard let decodedBase64Data = Data(base64EncodedURLSafe: base64EncodedString) else {
+            print(logTag,"Invalid base64 URL string provided")
+            throw DownloadFailedError.customError(description: "Error while base64 url decoding the credential response")
+        }
+        
+        let inputToCBORDecode = Array(decodedBase64Data)
+        let decodedCBORData = try! CBOR.decode(inputToCBORDecode)! as CBOR
```

Comment:
can we avoid `try!`

Reply:
we can refer this article `https://cocoacasts.com/what-is-the-difference-between-try-try-and-try`

Reply:
Thanks for pointing this out, have modified the code to reflect the same